packages/edge-gateway/package.json

```diff
@@ -11,11 +11,12 @@
     "predev": "node scripts/cli.js ipfs --start",
     "dev": "miniflare dist/worker.mjs --watch --debug -m",
     "postdev": "node scripts/cli.js ipfs --stop",
-    "test": "npm run test:setup && npm-run-all -p -r mock:ipfs.io mock:cf-ipfs.com test:worker",
+    "test": "npm run test:setup && npm-run-all -p -r mock:ipfs.io mock:cf-ipfs.com mock:cid-verifier.io test:worker",
```

Comment:
We shouldn't mock cid verifier like this here. If we use the worker binding (see my comment on wrangler.toml file), we should test using the worker binding miniflare API. For this you can look on how we test the gateway API binding:
- https://github.com/web3-storage/reads/blob/main/packages/edge-gateway/test/utils/miniflare.js#L24-L32
- https://github.com/web3-storage/reads/blob/main/packages/edge-gateway/test/utils/scripts/api.js

This is essentially the way Miniflare enables us to test Worker to Worker interactions. So, in this case, most tests should be in `cid-verifier` package. In edge-gateway, probably just the basic interactions with the external API